src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/WebSocketsTransport.cs

```diff
@@ -85,14 +90,18 @@ public WebSocketsTransport(HttpConnectionOptions httpConnectionOptions, ILoggerF
                     {
                         _webSocket.Options.UseDefaultCredentials = httpConnectionOptions.UseDefaultCredentials.Value;
                     }
-
-                    httpConnectionOptions.WebSocketConfiguration?.Invoke(_webSocket.Options);
                 }
 
+                httpConnectionOptions.WebSocketConfiguration?.Invoke(_webSocket.Options);
```

Comment:
should be in `if (!isBrowser)`

Reply:
I know it was that way before, but why not run the WebSocketConfiguration callback in the browser?

Reply:
I looked at ClientWebSocketOptions, and now it makes sense. I don't think there's a single thing there that could be configured in the browser. Maybe we should throw if it's set to a non-null value in the browser rather than jut quietly not call it.